### PR TITLE
Foods

### PR DESCRIPTION
**Pull request by podoglyph**
For deleting foods I went ahead and added a dependent: :destroy on the food model for the meal_food join table. I have a popup to have the user confirm that when deleting a food it will also be deleted from the meal tables. Is this the expected behavior? It didn't make sense to me to leave a food on the meal table if it is deleted from the food list.

This commit ought to have the relevant event listener on line:100 of event.js.
https://github.com/Benjaminpjacobs/quantified-self-client/commit/f73d324c2d3b48eee7e265f103907e33da1d1f77

### CODE REVIEW

File: lib/events.js [hunk 1]
```diff
@@ -98,10 +100,19 @@ $(document).ready(function() {
     $foodIndex.on('click', '.delete', function(event) {
         var food = this.parentElement.parentElement
         var id = food.id
-        Food.delete(id)
-            .then(function() {
-                $(`#${id}`).fadeOut(300);
-            })
+        Food.showPopUp(this, id);
+        //add event listener for click off popup
+        $(this).prev('.confirm-delete').on('click', 'button', function() {
```
Comment (Benjaminpjacobs) [inline lib/events.js:105]: nice delegation!

File: lib/food.js [hunk 2]
```diff
@@ -47,6 +47,17 @@ class Food {
         });
     }
 
+    static showPopUp(el, id) {
+        let popUp = `<div class='confirm-delete'>
```
Comment (Benjaminpjacobs) [inline lib/food.js]: this can probably be const, i don't see why it would have to change right?

File: lib/events.js (hunk 1 above)
Comment (case-eee) [reply lib/events.js:105]: 👍

File: lib/food.js [hunk 3]
```diff
@@ -11,8 +11,8 @@ class Food {
 
     toHTML() {
         return `<tr class='food' id=${this.id}>
-                  <td>${this.name}</td>
-                  <td class='calorie-count'>${this.calories}</td>
+                  <td contenteditable="true">${this.name}</td>
```
Comment (case-eee) [inline lib/food.js:14]: 👍

**Review by case-eee — COMMENTED** [review]
looks good to me @podoglyph 👍 

the dependent destroy is a good addition to the API as well 🎉

**Pull request merged**